### DISCUSSION TITLE
Number of keys in shortcut

### DISCUSSION THREAD

**Discussion by filiptammergard** [johnlindquist/kit#132] [Q&A] [answered]
Is there some limitation to the number of shortcut keys that can be used in a shortcut?

I have mapped caps lock to `cmd opt control shift` with Karabiner-Elements to be able to set new shortcuts in other programs without overriding something else, and I'm thinking about using that combination in Scriptkit as well. 

But when I try `// Shortcut: cmd opt shift control .` nothings happen independent on if I use caps lock or all the keys. Removing one of them solves it, though. Is there a limit of four keys?

**Comment by johnlindquist** [comment, marked as answer]
The shortcuts are a combination of the ["Accelerators" in Electron](https://www.electronjs.org/docs/api/accelerator#available-modifiers) and a [Shortcut Normalizer](https://github.com/johnlindquist/kitapp/blob/beta/src/shortcuts.ts#L13) I put together to make the syntax a little easier.

I tested `// Shortcut: cmd opt shift control .` and it successfully passes the `globalShortcut.register` test in Electron, but it doesn't do anything on my system either.

Instead of `.`, `z` worked for me:

```js
// Shortcut: cmd opt shift control z

await arg("long shortcut")
```

Does that work for you?

Maybe ` cmd opt shift control .` is some sort of reserved global shortcut on osx? Hmm...

**Reply by filiptammergard** [reply]
Yep, `cmd opt shift control z` works perfectly. Thanks!
